block-canvas/theme.json

```diff
@@ -343,7 +344,11 @@
             }
         },
         "spacing": {
-            "blockGap": "30px"
+            "blockGap": "1.5rem",
+            "padding": {
+                "right": "var(--wp--preset--spacing--30)",
```

Comment:
Do we need to opt in to spacing to get these?

Reply:
I see that they are coming through and it's working as expected without opting into anything.

However, *--spacing--30 seems a fair bit smaller than the original 30px.